src/strategies/ejs-staking/examples.json

```diff
@@ -0,0 +1,125 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "ejs-staking",
+      "params": {
+        "stakingServiceContractAddress": "0x9eF2a23292DC12A5597CcEB7F5039aF944896a9D",
+        "voteEndDate": "2022-7-1 2:22:22 GMT",
+        "decimals": 0,
+        "stakingPoolSettings": [
```

Comment:
Hi @terence-ejs Looks like any one can pass any number of arrays in here and into pools subArray which kills the 5 request limit, check this checklist 

https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

Reply:
Hi @ChaituVR you can pass any number of arrays into the pools subArray but the number of requests is capped by the number of networks configured in the EJS_SUBGRAPH_URL constant

Reply:
@terence-ejs In future someone may add their own network in EJS_SUBGRAPH_URL ?

Reply:
@ChaituVR Yes. We will add other networks in the future once we have deployed the contracts and subgraph. We will probably have just 2 networks configured, but if we have more than 5 networks configured I'm not really sure how we can keep to the 5 request limit. We're unable to use the multichain strategy as well as we want to aggregate the number of tokens cross network before calculating the votes.

Reply:
@ChaituVR can we have the pull request merged?

Reply:
hey @terence-ejs This strategy is quite big and as I said before, it can call more than 5 requests, which will not meet the checklist requirement https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy please find some other way to make it smaller and reduce the number of requests sent out.